### ISSUE TITLE
Trojan

### ISSUE THREAD

**Issue by Shubham123498765** [zhongyang219/TrafficMonitor#1352]
The latest version is flagged as trojan by virus total
![image](https://user-images.githubusercontent.com/82265932/194222868-857afcde-9540-4c80-9159-2d0563beb2b7.png)

**Comment by Coder-Aftab** [comment]
Yes after installing this all of my google accounts got logged out, my pc started slowing down and later when I checked the files with virus total its shows the same these people are making money by trojan

**Comment by MoXAriApph** [comment]
> Noticed the same and find it quite worrying. It might be a false positive of course, but the fact, that Google is among the red flags and also the reports are non-heuristic, is rather alarming. v1.83 (x64, lite) does not generate these warnings.

It seems like a false positive

**Comment by MoXAriApph** [comment]
@zhongyang219

**Comment by tunisiano187** [comment]
Same here : https://www.virustotal.com/gui/file/6b5d3091528e39f6f6f64f8725ab8bd4423eb9a6648706d4ad40bbfc7c0e3755/detection/f-6b5d3091528e39f6f6f64f8725ab8bd4423eb9a6648706d4ad40bbfc7c0e3755-1667569859
![image](https://user-images.githubusercontent.com/6095739/200131465-c3cc1df2-1dcc-404e-a696-c3c61dfc5efc.png)

**Comment by mictaege** [comment]
For the record: Since version 1.84 is flagged as a virus this version will not be approved by the chocolatey package manager.

https://community.chocolatey.org/packages/traffic-monitor/1.84

**Comment by tunisiano187** [comment]
> For the record: Since version 1.84 is flagged as a virus this version will not be approved by the chocolatey package manager.
> 
> https://community.chocolatey.org/packages/traffic-monitor/1.84

Indeed, that's the reason of my question ;-)

**Comment by jiatern** [comment]
Same here, but to my surprise V1.84_x86_Lite is not detected with any virus. x64 versions do.

**Comment by m2u-84** [comment]
@zhongyang219 is this getting any attention soon? This looks a bit much for random false positives…

**Comment by mictaege** [comment]
The new version 1.84.1 still has the same detections.

https://www.virustotal.com/gui/file/7217c298d56936c58e36f8b98fcea50537ae935d57bc85c946c6385b6ffdd7cc

![grafik](https://user-images.githubusercontent.com/4046390/202728594-86696c6d-ae4f-4843-abdf-0ebf7551eac7.png)

**Comment by roman6078** [comment]
Just built the latest version from master-branch: No detection on virus total.

**Comment by kylefmohr** [comment]
> Just built the latest version from master-branch: No detection on virus total.

I tried to replicate this and built from the master branch just a few minutes ago. 

For the x64 executable, [the full version has 0 detections](https://www.virustotal.com/gui/file/4f76d5ebf949d1824aa8e4686d543acc99bde204624605401b6624d0b9830ac1), the [Lite version now has a new one (Cylance), but only that one](https://www.virustotal.com/gui/file/1424158f67e66562584fbbe17af05f54d89b16ead8c78b890ee831793ffe1b60/detection). 

For the x86 executable, [the full version has a whopping **10 detections**](https://www.virustotal.com/gui/file/f1595f7e343ccc14023309cb2499e06842f6fc5c00f5c8969263facd6e8bf938/detection), and [the Lite version has the same one detection](https://www.virustotal.com/gui/file/a878944f688db040de5bf2ba122c5a1bc963f1d8e4cea74b27de1f18b6c8a9f8?nocache=1), from Cylance


I'm also here because of Chocolatey, do they require 0 VirusTotal detections to approve a package?

**Comment by 3xploiton3** [comment]
0 detection

https://www.virustotal.com/gui/file/2531cc7fb5c75b38b8715dca0c48ede7c1747d337422908e3f975df31d4f66f9
